pkg/bpf/mockbpf.go

```diff
@@ -1,17 +1,31 @@
-// Code generated by bpf2go; DO NOT EDIT.
-//go:build mips || mips64 || ppc64 || s390x
-
+//go:build !bpf
+// +build !bpf
+
+/*
+Copyright 2024.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
 package bpf
 
 import (
 	"bytes"
-	_ "embed"
 	"fmt"
 	"io"
 
 	"github.com/cilium/ebpf"
 )
 
+//nolint:stylecheck // This is just a dummy file
```

Comment:
Does this mean that we need to update the mock when the struct changes?
bpf2go autogenerates this for us.